site/docs-md/README.md

```diff
@@ -20,6 +20,7 @@
   * [Cordova/Ionic Native Plugins](cordova/using-cordova-plugins.md)
   * [Known Incompatible Plugins](cordova/known-incompatible-plugins.md)
 * Guides
+  * [Splash Screens and Icons](guides/splash-screens-icons.md)
```

Comment:
put it here, since I think it's the best spot for now. Created a dedicated page not only for SEO/people searching for it, but to make it stand out more in the docs menu bar.